### PR TITLE
add rental contact

### PR DESCRIPTION
**Pull request by hauska7**
https://bookingsync.atlassian.net/browse/CORE-1256

### CODE REVIEW

**Comment by adamgrad** [comment]
@hauska7 did you run the app locally and check if there are no UI issues?

File: api_calls/reference/rental_contacts/public-scope-response.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ "links": {
+    "rental_contacts.rental": "http://test.host/api/v3/rentals/{rental_contacts.rental}",
```
Comment (Azdaroth) [inline api_calls/reference/rental_contacts/public-scope-response.json]: the URLs should point to https://bookingsync.com

File: api_calls/reference/rental_contacts/public-scope-response.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+     "kind": "manager",
+     "roles": ["requests"],
+     "created_at": "2019-03-21T14:37:39Z",
+     "updated_at": "2019-03-21T14:37:39Z",
+     "canceled_at": null
+   }
+ ],
+ "meta": {
+   "Link": {
+     "first": "http://test.host/api/v3/rental_contacts?page=1",
```
Comment (Azdaroth) [inline api_calls/reference/rental_contacts/public-scope-response.json]: the URLs should point to https://bookingsync.com

File: api_calls/reference/rental_contacts/rentals_read-rentals_write-scope-response.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ "links": {
+    "rental_contacts.rental": "http://test.host/api/v3/rentals/{rental_contacts.rental}",
```
Comment (Azdaroth) [inline api_calls/reference/rental_contacts/rentals_read-rentals_write-scope-response.json]: the URLs should point to https://bookingsync.com

File: content/reference/endpoints/rental_contacts.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Returns a newly created rental contact.
+
+~~~~
+POST /rental_contact
+~~~~
+
+<%= render 'json_response', endpoint: "rental_contacts", request: "create",
+  scopes: [{ rentals_write: "rentals_read-rentals_write" }] %>
+
+## Update a rental
```
Comment (Azdaroth) [inline content/reference/endpoints/rental_contacts.md]: rental contact, no?

File: content/reference/endpoints/rental_contacts.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Returns an updated rental identified by ID.
```
Comment (Azdaroth) [inline content/reference/endpoints/rental_contacts.md]: rental contact, no?

File: content/reference/endpoints/rental_contacts.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  <div class="tab-pane active" id="public" markdown="1">
+Name                            | Type     | Read/Write | Description
+--------------------------------|----------|------------|------------
+id                              | Integer  | Read       | Rental contact's id.
+contact_id                      | Integer  | Read       | Contact's id.
+--------------------------------|----------|------------|--------------
+kind                            | [String](/reference/enums#rental-contact-kinds) | Read | Rental contact's kind
+roles                           | [Array of Strings](/reference/enums#rental-contact-roles) | Read | Rental contact's roles
+--------------------------------|----------|------------|--------------
+created_at                      | [Time](/reference/enums#formats) | Read         | Rental's create time.
```
Comment (Azdaroth) [inline content/reference/endpoints/rental_contacts.md]: rental contact, no?

File: content/reference/endpoints/rental_contacts.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List rental contacts
+
+List all rental contacts for a given account(s).
+
+~~~
+GET /rental_contacts
```
Comment (Azdaroth) [inline content/reference/endpoints/rental_contacts.md]: should be prefixed by /api/v3, same everywhere

File: content/reference/endpoints/rental_contacts.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+--------------------------------|----------|------------|------------
+id                              | Integer  | Read       | Rental's id.
+contact_id                      | Integer  | Write      | Contact's id.
+--------------------------------|----------|------------|--------------
+kind                            | [String](/reference/enums#rental-contact-kinds) | Read/Write | Rental contact's kind
+roles                           | [Array of Strings](/reference/enums#rental-contact-roles) | Read/Write | Rental contact's roles
+--------------------------------|----------|------------|--------------
+created_at                      | [Time](/reference/enums#formats) | Read         | Rental's create time.
+updated_at                      | [Time](/reference/enums#formats) | Read         | Rental's update time.
+canceled_at                     | [Time](/reference/enums#formats) | Read         | Rental's canceled time if canceled.
```
Comment (Azdaroth) [inline content/reference/endpoints/rental_contacts.md]: rental contact, no? same everywhere

File: api_calls/reference/rental_contacts/public-scope-response.json (hunk 1 above)
Comment (hauska7) [reply api_calls/reference/rental_contacts/public-scope-response.json]: good catch!

File: content/reference/endpoints/rental_contacts.md (hunk 7 above)
Comment (hauska7) [reply content/reference/endpoints/rental_contacts.md]: actually this is the same as other endpoints have. Eg: http://developers.bookingsync.com/reference/endpoints/rentals/#list-rentals

Comment (Azdaroth) [reply content/reference/endpoints/rental_contacts.md]: Hmm, ok, so let's keep it like this for now

**Comment by hauska7** [comment]
Yes I run locally and it works fine. I think after merge we need to `rake publish` and also notify someone as in jira

**Comment by hauska7** [comment]
@Azdaroth all feedback applied. Is good to merge.

**Comment by Azdaroth** [comment]
@adamgrad can you please take over the review and release if all is good here?

**Comment by thilonel** [comment]
@adamgrad

**Pull request merged**